provision/provision.sh

```diff
@@ -452,14 +452,6 @@ tools_install() {
   # Disable xdebug before any composer provisioning.
   sh /srv/config/homebin/xdebug_off
 
-  echo " * Checking for NVM"
-  if [[ -f ~/.nvm ]]; then
```

Comment:
Wrap this with `if is_utility_installed core nvm; then`

Reply:
You mean this

```bash
echo " * Checking for NVM"
  if [[ ! is_utility_installed core nvm && -f ~/.nvm ]]; then
    echo " * .nvm folder found, switching to system node, and removing NVM folders"
    nvm use system
    rm -rf ~/.nvm ~/.npm ~/.bower /srv/config/nvm
    echo " * NVM folders removed"
fi
```

Reply:
We could, but I'd rather implement an official method of removal for utilities than add cleanup code for every utility to core. We should merge this as is

Reply:
@tomjn In that case maybe have every utility add a `uninstall.sh` which could be run to remove that utility? Because every utility would have different un-installation since installation isn't standard for all of them.